vue-frontend/src/pages/portfolio/index.vue

```diff
@@ -6,19 +6,104 @@
       </template>
     </metainfo>
     <Header />
-    <Portfolio />
-    <CTA />
+    <div
```

Comment:
You can take the whole portfolio section from the home page. Just replace CTA section based on URL.